src/ck-core/debug-charm.C

```diff
@@ -932,8 +953,14 @@ void CpdRemoveBreakPoint (char *msg)
         if (breakPointEntryInfo != NULL) {
           if (--breakPointEntryInfo->msgIdx == 0) {
             // If we are the last to delete the breakpoint, then restore the original name and call function pointer
-            _entryTable[idx]->name =  breakPointEntryInfo->name;
-            _entryTable[idx]->call = (CkCallFnPtr)breakPointEntryInfo->call;
+#if CMK_SMP	    
+	    if(_entryTable[idx]->breakPointSet--==0)
```

Comment:
Is postfix decrement here and and on line 987 correct?

Reply:
Good catch.  Didn't really test removing breakpoints until you pointed that out.  in !fixup